src/screens/Exchange/utils.js

```diff
@@ -200,7 +200,7 @@ export const getHeaderRightItems = (
 const isEnoughAssetBalance = (assetBalance: ?string, amount: string): boolean => {
   try {
     const amountBN = new BigNumber(amount);
-    const balanceBN = new BigNumber(assetBalance);
+    const balanceBN = new BigNumber(assetBalance ?? 0);
```

Comment:
This change assumes that missing asset balance is equal to balance of 0 which will lead to returning false for all non-zero `amount` values. Previously this was BigNumber(NaN)